modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -733,9 +758,11 @@ definition {
 				panel = "Control Panel",
 				portlet = "Queue");
 
-			AssertTextPresent(value1 = "edituserln");
+			AssertTextPresent(value1 = "userfn");
 
 			AssertTextPresent(value1 = "userea@liferay.com");
+
+			AssertTextPresent(value1 = "After");
```

Comment:
If the test description says __**"terms were filled in correctly for the To, From Address, From Name, CC, BCC, and Template Body fields"**_ why aren't we validating CC, BCC, and email body fields? could we remove this detail from the description?

Reply:
Since the verification if the terms are replaced correctly are did on the QUEUE, and in the QUEUE we aren't able to see the **CC** and **BCC** and **email body** fields, don't make sense we put the terms in these fields, you agree?

Reply:
Got it, but that's what I mean because in the description we are saying that we are going to fill these fields but in the test, we are not filling them

Reply:
Done!